browser/autocomplete/brave_autocomplete_provider_client.cc

```diff
@@ -39,3 +39,14 @@ std::vector<base::string16> BraveAutocompleteProviderClient::GetBuiltinURLs() {
   }
   return v;
 }
+
+bool BraveAutocompleteProviderClient::IsOffTheRecord() const {
+  return profile_->IsOffTheRecord();
+}
+
+void BraveAutocompleteProviderClient::StartServiceWorker(
+    const GURL& destination_url) {
+  if (profile_->IsOffTheRecord())
```

Comment:
I'm confusing with this. Is it possible that `ChromeAutocompleteProviderClient::profile_` is OTR?
It's always initialized by `ChromeAutocompleteProviderClient(profile->GetOriginalProfile())`